src/NBitcoin.Tests/ChainTests.cs

```diff
@@ -461,20 +454,123 @@ public ChainedBlock AppendBlock(ChainedBlock previous, params ConcurrentChain[]
         {
             ChainedBlock last = null;
             var nonce = RandomUtils.GetUInt32();
-            foreach(var chain in chains)
+            foreach (var chain in chains)
             {
                 var block = TestUtils.CreateFakeBlock(new Transaction());
                 block.Header.HashPrevBlock = previous == null ? chain.Tip.HashBlock : previous.HashBlock;
                 block.Header.Nonce = nonce;
-                if(!chain.TrySetTip(block.Header, out last))
+                if (!chain.TrySetTip(block.Header, out last))
                     throw new InvalidOperationException("Previous not existing");
             }
             return last;
         }
+
         private ChainedBlock AppendBlock(params ConcurrentChain[] chains)
         {
             ChainedBlock index = null;
             return AppendBlock(index, chains);
```

Comment:
missing `this.` many times in this file, not only here

Reply:
I didn't put any effort into cleaning this file other than just adding my new tests to the end (as it seemed like the most logical place for them). Not sure what the plan is for the NBitcoin tests longer term?

Reply:
aha, this is NBitcoin.Tests, chmm ... no idea :)

Reply:
I would like to remove (eventually) NBitcoin tests, either delete them or move them to the appropriate place in the FN.

Reply:
Ok we should probably have a discussion on moving ChainedBlock, ConcurrentChain and ChainBase out of NBitcoin in a future PR. We can discuss in #659

Reply:
Ok I assume the tests will move together with the class and so it is worth applying styling to it (maybe not this PR)